example-llm-workflows/README.md

```diff
@@ -0,0 +1,483 @@
+
+So, what does that mean for us? If we provide the same arguments to our Guest Wasm functions, Homestar will return cached responses instantly, which saves us both time and resources, as waiting for large prompt responses when using models, even *smaller* ones like LLaMA 3 trained with 8 billion parameters, can be somewhat latent on devices with limited GPU and memory (like my personal computer).
+
+Is there a way to always provide *new* hallucinations from the Host model? Yes, there is! We can use the `--debug` flag when we start EveryCli to force re-execution of the tasks in our workflow.
+
+```console
+./every-cli/cli.js dev ../example-llm-workflows/llm.wasm --debug
+```
+
+## Foundations for Privacy
+
+The learning goals of this project were to experiment with working with LLMs locally on hosts where the training data and tuning of a model remains private and only derived information from prompt-based AI chains can be shared with other users/peers for consumption, allowing for working with AI computation in ways not tied to any specific vendor or large cloud provider. Essentially, this showcases a positive avenue for decentralized, user-controlled AI-oriented computation that's everything that [IEEE Spectrum's Open-Source AI Is Uniquely Dangerous][ieee] isn't.
+
+Localized, open-source LLM models for home and on-prem use cases are growing in popularity, even leading to the creation of a [LocalLLaMA reddit][reddit-post] community! We've seen how [GDPR][gdpr] has increased the need for companies to be more careful around [PII management and data privacy isolation across regions and region-compliance laws][so-privacy].
+
+Self-hosted, privately managed model deployments hit on many of the privacy and security modules taught in our course. Incorporating ways for users to chain LLM steps together while controlling what inference gets exhibited without the infrastructure concerns or data risks typically associated with external cloud services, presents a unique opportunity to democratize AI capabilities. By ensuring that users can interact with and execute complex AI
+workflows with ease, the project aims to bridge the gap between advanced AI technologies and those with some software development background. This approach not only aligns with the course's focus on privacy and security, but also empowers users by providing them with tools to leverage AI in a secure, private, and user-friendly manner.
+
+Regarding further alignment with our class, the rise of LLMs has made the transparency of existence, operation, and first-class design for transparency urgent ethical matters as more and more companies move to gather data and push AI agents into our everyday lives. OpenAI's chat interface (to GPT) spawned much of the burgeoning popularity of LLMs, but with OpenAI, we know the CEO of the company is linked to another company (Worldcoin) that has [shady digital-identity practices][iris-scanning].
```

Comment:
"transparency" twice here is confusing.

Reply:
@bgins this actually hits on bullets in a lecture :).